### PR TITLE
Home page fixed

### PR DESCRIPTION
**Pull request by xhhh-mendes**
please review

### CODE REVIEW

**Review by IvanImp — CHANGES_REQUESTED** [review]
Good work, but there are some issues that you can look into
1. The picture you showed in messenger, the "Gas" label overlapped with "Transport" with no color on it
2. The "Set Emission Goal" can look more like a button rather than label
3. Set Emission Goal popup can gives more hint like "Emission Goal (kg) ____________"
4. Color of the "Save" can try emphasisColor
5. However I'm not sure about the donut-shaped graph, maybe someone can give more insight on it.

Thanks for your work anyway :) Screenshot can leave it here in PR instead of messenger when update

File: pubspec.yaml [hunk 1]
```diff
@@ -94,5 +94,12 @@ flutter:
   #       - asset: fonts/TrajanPro_Bold.ttf
   #         weight: 700
   #
+  fonts:
+    - family: Raleway
```
Comment (pu-pu) [inline pubspec.yaml:98]: Are you using this font? Couldn't find any reference.

**Review by pu-pu — CHANGES_REQUESTED** [review]
Mostly the same issues as Ivan mentioned. One additional point regarding the alarm clock icon on the second screenshot. I don't understand that symbol. Is it the set emission goals button?

**Review by RW21 — CHANGES_REQUESTED** [review]
Thanks for the PR. 😄 
I think the donut shape graph is fine.
I get errors when there are no entries in some emission category.
```
The method '[]' was called on null.
```

**Comment by xhhh-mendes** [comment]
1. As Gregoria said, self-adding emission goal is disabled for now.

2. There is no 'Gas' section in the pie chart because gas input hasn't been done.

3. The error @RW21 mentioned, the error occurs even there are entries. I couldn't figure it out, but it seems doesn't affect functionality.

File: lib/scenes/home_page.dart [hunk 2]
```diff
@@ -14,35 +17,7 @@ class Homepage extends StatefulWidget{
...
-    );
-  }
-
-  @override
-  void initState(){
-    super.initState();
-    _seriesPieData = List<charts.Series<Task, String>>();
-    _generateData();
-  }
+  String _emissionGoal = '1000 KG/CO\u2082';
```
Comment (RW21) [inline lib/scenes/home_page.dart:20]: Please get sum of today's emission.

**Review by RW21 — CHANGES_REQUESTED** [review]
Thanks for the fixes 😄 
- Also, can you add text if there is no emission for the day? 

Actually, it works.
~~Can you also try to fix getting the home energy data in the pie chart? (I think something with source_id is broken because my `source_id` for `Home Energy` was 43. )~~
~~or do you think we should do this in another task?~~

**Review by IvanImp — APPROVED** [review]
Is there a need to split up home energy into micro parts (gas, electricity, water). Maybe someone else could give their thoughts on this as well. In the mean time, we could merge this first and the splitting can be another feature. Else, LGMT. Thanks for the work

**Review by pu-pu — APPROVED** [review]

**Comment by xhhh-mendes** [comment]
@RW21
Rather than directly display text, I still use a pie chart which looks like this.
no
Does it look good to you?

**Comment by xhhh-mendes** [comment]
@RW21 I forgot to push lol, I'll push it first

**Review by RW21 — APPROVED** [review]
Thanks 😄

**Pull request merged**